### ISSUE TITLE
Babili no longer working with webpack 3.0.0

### ISSUE THREAD

**Issue by bradennapier** [babel/babili#590]
As of webpack 3 this appears to no longer work.  I thought it was a few other things but I have narrowed it down to babili breaking arg spreading.

I have updated everything to the latest.  

```js
const r = Object.keys(selected).reduce((...args) => {
    console.log(args.length); // 4
    handleProcessReduction(selected, ...args);
  }, {});

function handleProcessReduction(selected, ...args) {
  console.log('p: ', args.length); // 1
} 
```

I can see it is not doing the right thing:

```js
return Object.keys(e).reduce(function() {
            for (var r = arguments.length, t = Array(r), n = 0; n < r; n++)
              t[n] = arguments[n];
            console.log(t.length), u.apply(void 0, [e, t]); // [e, t] not [e].concat(t)?
          }, {});
```

My babel config is just default

```js
babel: {
    plugins: ['transform-class-properties'],
    presets: [
      'stage-0',
      [
        'env',
        {
          modules: false,
        },
      ],
    ],
    env: {
      production: {
        presets: ['babili'],
      },
      test: {
        plugins: ['istanbul'],
        sourceMaps: 'inline',
      },
    },
  },
```

**Comment by bradennapier** [comment]
It works perfectly in the reply which adds to the annoyance here haha

I'm sure it's something stupid I am doing but I can't imagine what - this is an extremely simple setup.

**Comment by goto-bus-stop** [comment]
Yeah, this one has been biting a lot of people—it's fixed on master by #578, but not yet released!

**Comment by vigneshshanmugam** [comment]
Released new version https://github.com/babel/babili/releases/tag/babili%400.1.4
